client/rpcserver/handlers.go

```diff
@@ -399,19 +399,20 @@ func handleExchanges(s *RPCServer, _ *RawParams) *msgjson.ResponsePayload {
 	return createResponse(exchangesRoute, &exchanges, nil)
 }
 
-// handleLogin sets up the dex connection and returns core.LoginResult.
+// handleLogin sets up the dex connections.
 func handleLogin(s *RPCServer, params *RawParams) *msgjson.ResponsePayload {
 	appPass, err := parseLoginArgs(params)
 	if err != nil {
 		return usage(loginRoute, err)
 	}
 	defer appPass.Clear()
-	res, err := s.core.Login(appPass)
+	err = s.core.Login(appPass)
```

Comment:
As far as I can tell, this leaves no way to get notifications through the RPC server.

Reply:
New `notifications` command?

Reply:
I'm not convinced its needed for the RPC server.  Even headless Go consumers likely just dump the notifications.